command.py

```diff
@@ -181,7 +191,7 @@ async def unban(self, interaction: discord.Interaction, member: discord.User,
                                                                            interaction.guild.name
                                                                            ))
 
-class logChannelSelect(app_commands.Group):
+class log_channel_select(app_commands.Group):
```

Comment:
Sorry!! IT IS MY MISTAKE!!
Please, apply this suggestion
```suggestion
class logChannelSelect(app_commands.Group):
```